### PR TITLE
[electrum] Add test for unknown method

### PR DESCRIPTION
**Pull request by dagurval**
Test for https://github.com/BitcoinUnlimited/ElectrsCash/pull/36

### CODE REVIEW

File: qa/rpc-tests/test_framework/electrumutil.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ def bitcoind_electrum_args():
     ELECTRUM_PORT = random.randint(40000, 60000)
     return ["-electrum=1", "-debug=electrum", "-debug=rpc",
             "-electrum.port=" + str(ELECTRUM_PORT),
+            "-electrum.exec=/home/dagurval/ElectrsCash/target/debug/electrscash",
```
Comment (sickpig) [inline qa/rpc-tests/test_framework/electrumutil.py]: this change seems unrelated

Comment (dagurval) [reply qa/rpc-tests/test_framework/electrumutil.py]: fixed

**Comment by dagurval** [comment]
Also added a test for https://github.com/BitcoinUnlimited/ElectrsCash/pull/38

File: qa/rpc-tests/test_framework/electrumutil.py (hunk 1 above)
Comment (dagurval) [reply qa/rpc-tests/test_framework/electrumutil.py]: removed

**Review by sickpig — APPROVED** [review]
